docs/sources/view-and-analyze-profile-data/profiling-types/_index.md

```diff
@@ -75,29 +75,29 @@ This is a clear indicator of a memory leak.
 
 ![memory leak example](https://grafana.com/static/img/pyroscope/pyroscope-memory-leak-2023-11-30.png)
 
-Without profiling, this is may be something that's exhibited in metrics or out-of-memory errors (OOM) logs but with profiling you have more details into the specific function that's allocating the memory which is causing the leak at the line level.
+Without profiling, this is maybe something that's exhibited in metrics or out-of-memory errors (OOM) logs but with profiling you have more details into the specific function that's allocating the memory which is causing the leak at the line level.
```

Comment:
Changing the wording. Do you think "this may be..." or "this is..." works better in this case? Feel free to edit this comment to update the suggestion. 
```suggestion
Without profiling, this may be something that's exhibited in metrics or out-of-memory errors (OOM) logs but with profiling you have more details into the specific function that's allocating the memory which is causing the leak at the line level.
```